xbmc/music/tags/TagLoaderTagLib.cpp

```diff
@@ -462,6 +464,44 @@ bool CTagLoaderTagLib::ParseTag(ID3v2::Tag *id3v2, EmbeddedArt *art, MUSIC_INFO:
           tag.SetUserrating(POPMtoXBMC(popFrame->rating()));
         }
       }
+    else if (it->first == "CHAP")
+    {
+      //We use a chapter number index here to ensure chapters are added in the right order to the map
+      int chapNumber = 0;
+      unsigned int prevTime = 0;
+      for (ID3v2::FrameList::ConstIterator ch = it->second.begin(); ch != it->second.end(); ++ch)
+      {
+        auto* chapFrame = dynamic_cast<ID3v2::ChapterFrame*>(*ch);
+        if (!chapFrame)
+          continue;
+        std::string embeddedTitle{};
```

Comment:
`{}`can be omitted, default ctor will be called automatically to init the string.

Reply:
Done.